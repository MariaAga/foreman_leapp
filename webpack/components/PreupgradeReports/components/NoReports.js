import React from 'react';

import { EmptyStatePattern as EmptyState } from 'foremanReact/components/common/EmptyState';
import { translate as __ } from 'foremanReact/common/I18n';

const NoReports = () => (
  <EmptyState
    iconType="pf"
    icon="in-progress"
    header={__('No Preupgrade Reports Found')}
    description={__(
      'If job execution is still running, reports may not have been yet created. If job generates any reports, they will be loaded once the job finishes.'
    )}
  />
);

export default NoReports;
