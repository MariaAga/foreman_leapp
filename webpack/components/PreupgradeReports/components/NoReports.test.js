import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import NoReports from './NoReports';

const fixtures = {
  'should render': {},
};

describe('NoReports', () =>
  testComponentSnapshotsWithFixtures(NoReports, fixtures));
