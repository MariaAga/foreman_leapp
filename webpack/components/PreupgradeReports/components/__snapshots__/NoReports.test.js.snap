// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoReports should render 1`] = `
<EmptyStatePattern
  description="If job execution is still running, reports may not have been yet created. If job generates any reports, they will be loaded once the job finishes."
  header="No Preupgrade Reports Found"
  icon="in-progress"
  iconType="pf"
/>
`;
